clients/studio/plugins/ThymioBlockly/ThymioBlockly.h

```diff
@@ -39,97 +39,101 @@
 
 #include "AsebaJavascriptInterface.h"
 
-namespace Aseba { namespace ThymioBlockly
```

Comment:
This namespace change breaks all the code, I think it is not worth it, can we disable this style constraint?